### ISSUE TITLE
Attribute macro items not detected if there's no trivia at the end and start

### ISSUE THREAD

**Issue by wawel37** [software-mansion/cairo-lint#370]
### Problem

If there's missing trivia at the start and end of the item (newlines etc.), the linter won't give any diagnostics here:

```rust
#[simple_attribute_macro_v2]
fn foo() {
    let a = 1;
    let b = 2;
    if a == 1 {
        if b == 2 {
            println!("a is 1 and b is 2");
        }
    }

    // test comment
    let _should_lint = (1 == 1);
}
```

### Scarb Version

```text
2.11.4
```
